YogaKit/YKLayout.h

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "YKEnums.h"
+
+@interface YKLayout : NSObject
+
+/**
+ The property that decides if we should include this view when calculating layout. Defaults to YES.
+ */
+@property (nonatomic, setter=setIncluded:) BOOL isIncluded;
+
+/**
+ The property that decides during layout/sizing whether or not yk_* properties should be applied. Defaults to NO.
+ */
+@property (nonatomic, setter=setEnabled:) BOOL isEnabled;
+
+@property (nonatomic) YKDirection direction;
+@property (nonatomic) YKFlexDirection flexDirection;
+@property (nonatomic) YKJustify justifyContent;
+@property (nonatomic) YKAlign alignContent;
+@property (nonatomic) YKAlign alignItems;
+@property (nonatomic) YKAlign alignSelf;
+@property (nonatomic) YKPositionType positionType;
+@property (nonatomic) YKWrap flexWrap;
+
+@property (nonatomic) CGFloat flexGrow;
+@property (nonatomic) CGFloat flexShrink;
+@property (nonatomic) CGFloat flexBasis;
+
+@property (nonatomic) CGFloat positionLeft;
+@property (nonatomic) CGFloat positionTop;
+@property (nonatomic) CGFloat positionRight;
+@property (nonatomic) CGFloat positionBottom;
+@property (nonatomic) CGFloat positionStart;
+@property (nonatomic) CGFloat positionEnd;
+@property (nonatomic) CGFloat positionHorizontal;
```

Comment:
@hartbit Wow, I didn't even notice those. Do we think they need to part of `YogaKit`? I purposely left some properties out at first because I didn't see a huge use-case for them with UIKit.